services/brig/src/Brig/Data/Instances.hs

```diff
@@ -287,3 +287,10 @@ instance Cql SearchVisibilityInbound where
   fromCql (CqlInt 0) = pure SearchableByOwnTeam
   fromCql (CqlInt 1) = pure SearchableByAllTeams
   fromCql n = Left $ "Unexpected SearchVisibilityInbound: " ++ show n
+
+instance Cql (Imports.Set BaseProtocolTag) where
+  ctype = Tagged IntColumn
+
+  toCql = CqlInt . fromIntegral . protocolSetBits
+  fromCql (CqlInt bits) = pure $ protocolSetFromBits (fromIntegral bits)
```

Comment:
If I'm reading this right, we simply ignore any bits other than the lowest two. E.g., if value 5 is read from the DB, this is fine and in particular we'll say that this means that Proteus is supported (because 5 is 101 in binary, and the lowest bit is set, which corresponds to Proteus).

Reply:
That's right.